pythonosc/osc_server.py

```diff
@@ -138,15 +143,36 @@ class ForkingOSCUDPServer(
   and forking a whole new process for each of them is worth it.
   """
 
-  def __init__(self, server_address, dispatcher):
-    super().__init__(server_address, _UDPHandler)
+
+class AsyncIOOSCUDPServer():
+  """Asyncio version of the OSC UDP Server.
+
+  :param server_address: tuple of (IP address to bind to, port)
+  :param dispatcher: a pythonosc.dispatcher.Dispatcher
+  :param loop: an asyncio event loop
+
+  Each UDP message is handled by _handle, the same method as in the OSCUDPServer family of blocking, threading, and forking servers
+  """
+
+  def __init__(self, server_address, dispatcher, loop):
+    self._server_address = server_address
     self._dispatcher = dispatcher
+    self._loop = loop
 
-  def verify_request(self, request, client_address):
-    """Returns true if the data looks like a valid OSC UDP datagram."""
-    return _is_valid_request(request)
+  class OSCProtocolFactory(asyncio.DatagramProtocol):
+    """OSC protocol factory which passes datagrams to _handle"""
+
+    def __init__(self, dispatcher):
+      self.dispatcher = dispatcher
+
+    def datagram_received(self, data, addr):
+      _handle(data, self.dispatcher)
+
+  def serve(self):
+    """creates a datagram endpoint and registers it with our event loop"""
+    listen = self._loop.create_datagram_endpoint(lambda: self.OSCProtocolFactory(self.dispatcher), local_addr=self._server_address)
```

Comment:
move the lambda on the previous line with 4 spaces indent to fit in 80 chars please